src/lib/pages/stored-codes/index.tsx

```diff
@@ -77,16 +82,25 @@ const StoredCodes = observer(() => {
             {storedCodesCount}
           </Badge>
         </Flex>
-        <Flex gap={2}>
+        <Skeleton
+          isLoaded={!isUploadAccessFetching}
+          display="flex"
+          gap={2}
+          borderRadius={8}
+        >
           {isPermissionedNetwork ? (
             <>
               <UploadButton isAllowed={isAllowed} />
-              <ProposalButton />
+              {govConfig.enabled &&
+                !(
+                  govConfig.disableStoreCodeProposal ||
+                  govConfig.disableOpenProposal
+                ) && <ProposalButton />}
```

Comment:
I think checking for `.enabled` is not necessary

Reply:
As I answered above